.github/workflows/publish_github_release_notes.yml

```
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This workflow will take the release notes from 
+# /website/www/site/content/en/blog/beam-${RELEASE_VERSION}.md" 
+# and updated in beam repo. 
+
+name: Publish Github Release Notes
+
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE_VERSION:
+        description: Release Version to update notes into beam repo 
+        required: true
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3        
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  publish_github_release_notes:
+    runs-on: [self-hosted, ubuntu-20.04]
+    needs: set-properties
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RELEASE_VERSION: ${{ github.event.inputs.RELEASE_VERSION}}
+    name: Publish Github Release Notes
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Publish github release notes
+        run: |
+          POST_PATH=$(echo "website/www/site/content/en/blog/beam-${{env.RELEASE_VERSION}}.md")
```

```suggestion
          POST_PATH="website/www/site/content/en/blog/beam-${{env.RELEASE_VERSION}}.md"
```

Any reason to nest this in an echo?

---

Not really, deleted in the last commit